app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
We can add some description to the schemas actually since JSON schemas are designed to be self-descriptive (and there is a `"description"` key for comments). But I'm not sure that everybody who wants to find this information will manage to guess it's there.